docs/faq/data-loading.md

```diff
@@ -53,24 +68,34 @@ Unfortunately not yet.
 
 ## **Do you support change data capture \(CDC\) or logical replication for databases?**
 
-Airbyte currently supports [CDC for Postgres and Mysql](../understanding-airbyte/cdc.md). Airbyte is adding support for a few other databases you can check in the roadmap.
+Airbyte currently supports [CDC for Postgres and Mysql](../understanding-airbyte/cdc.md). Airbyte is adding support for a few other 
+databases you can check in the roadmap.
 
 ## Using incremental sync, is it possible to add more fields when some new columns are added to a source table, or when a new table is added?
 
 For the moment, incremental sync doesn't support schema changes, so you would need to perform a full refresh whenever that happens.
 Here’s a related [Github issue](https://github.com/airbytehq/airbyte/issues/1601).
 
+## There is a limit of how many tables one connection can handle?
+
+Yes, for more than 6000 thousand tables could be a problem to load the information on UI.
```

Comment:
link this to the new-connection section? centralising this makes it easier to maintain.

something like `Yes. Please see the this section for more information.'